pubspec.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^1.0.6
+  cupertino_icons:
   sqflite:
```

Comment:
can we specify the versions for the dependencies.

Reply:
Is it really need to specify the version?